packages/website/src/routes/daisy/tooltip/index.tsx

```diff
@@ -1,5 +1,20 @@
 import { component$ } from '@builder.io/qwik';
-import { Tooltip } from '@qwik-ui/theme-daisy';
+import { Tooltip, TooltipProps } from '@qwik-ui/theme-daisy';
+
+export const positions: TooltipProps['position'][] = [
```

Comment:
Why not to use a TS enum instead?
`export enum positions { 
   TopEnd = 'top-end', 
   Top = 'top',
   ...
}`

Reply:
They are already typed with the FloatingUI Placement enum.